openshift_scalability/content/pod-stress-simple.json

```diff
@@ -0,0 +1,37 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "stress-pod",
+    "creationTimestamp": null,
+    "labels": {
+      "name": "stress"
+    }
+  },
+  "spec": {
+    "containers": [
+      {
+        "name": "stress",
+        "image": "progrium/stress",
+        "ports": [
+          {
+            "containerPort": 8080,
+            "protocol": "TCP"
+          }
+        ],
+        "command": ["stress", "--cpu", "2", "--io", "1", "--vm", "2", "--vm-bytes", "128M"],
```

Comment:
As-is, this will put 2 full cores of load, that's not going to work in our infra.  There are ~2000 cores total.  The purpose of stress is to put "some data" into prometheus around cpu and memory, not to peg all cores.

So I would suggest using a different version of this.  The 'stress-ng' package (from EPEL) allows for fractional CPU load to be put on. For example, this will put 5% CPU load per pod.

```stress-ng --cpu 1 --cpu-load 5```

Reply:
See also:  https://github.com/openshift/svt/tree/master/openshift_scalability/content/centos-stress

That container did become a bit of a dumping ground for stress tools (jmeter, mb, stress).  Maybe create a new first-class stress-ng pod for cpu/memory/oom/etc testing.